File: RealEstate.Domain/Roles/Manager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RealEstate.Domain.Roles
{
    public class Manager : Realtor
    {
        public List<Realtor> Team { get; private set; } = new List<Realtor>();

        public Manager(string name): base(name) { }
    }
}

File: RealEstate.Presentation/RealEstateProject/RealEstateProject/src/app/models/roles/manager/manager.ts
import { Realtor } from "../realtor/realtor";


export class Manager extends Realtor {
  team: Realtor[] = [];

  constructor(){

    super()
    this.id = '';
    this.realtorName = '',
    this.team = [],
    this.name = '',
    this.email = '';
  }
}
